### PR TITLE
Feature/farm 69 view chat groups

### PR DESCRIPTION
**Pull request by LeeMatthewHiggins**
[FARM-69 - Chat Group]

#### 📲 What

Added chat group screen

#### 🤔 Why
		
MVP requirement
		
#### 🛠 How
		
Added a content link concept to Article
Made parsing accept the Chat Group format from firebase
Reused Article list and Article detail widgets to render.

#### 👀 See
		
Screenshots / external resources
		 
#### ✅ Acceptance criteria

- [ ] Design Review for UI with BA completed. 
- [ ] Manually tested and verified on Android device/emulator (min. Lollipop 5.1)
- [ ] Showcase video / automation test
- [ ] Passing all tests
- [ ] Rebased/merged with latest changes from development and re-tested
- [ ] Removed unsed comments. TODOs must have JIRA for future work.

#### Coding Standards Checklist
- [ ] unit tests 80% coverage on testable code (functions, methods, class)
- [ ] (Architecture) redux state, reducers, actions, middleware defined under lib/redux
- [ ] ui elements defined under lib/ui. 
- [ ] Strings ready for localisation (e.g. defined in lib/utils/strings.dart)
- [ ] Assets (images/icons path) defined in lib/utils/assets.dart
- [ ] Colors defined in lib/utils/colors.dart
- [ ] Dimensions (ie margins padding) defined in lib/utils/dimens.dart
- [ ] TextStyles defined in lib/utils/styles.dart

Recommended Style Guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

#### 🕵️‍♂️ How to test

Notes for QA

#### Reviewers

@farmsmart/amido @farmsmart/wamf

### CODE REVIEW

File: lib/ui/recommendations/RecommentationsList.dart [hunk 1]
```diff
@@ -240,7 +240,7 @@ class RecommendationsList extends StatelessWidget implements ListViewSection {
   }) {
     Navigator.of(context).push(
       MaterialPageRoute(
-        builder: (context) => CropDetail(provider: provider),
+        builder: (context) => CropDetail(provider: provider), //TODO: LH add the recommedations detail header here (add to plot function)
```
Comment (albertmiro) [inline lib/ui/recommendations/RecommentationsList.dart:243]: That would be done in another PR?

**Review by davidfranquet — APPROVED** [review]

**Review by albertmiro — APPROVED** [review]
